lucifer/GUI/GUI.py

```diff
@@ -112,3 +115,16 @@
                                      font=self.font)
         if self.toolbar.isSettingsOpen:
             self.toolbar.settings_window.update_font()
+
+    def load_settings(self):
+        if Settings.is_settings():
+            font_settings = Settings.get_setting("gui.font")
+            try:
+                self.font = (font_settings["name"], font_settings["size"])
+                self.update_font()
+            except TypeError as e:
```

Comment:
Codacy found an issue: [local variable 'e' is assigned to but never used (F841)](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6191047)